packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -1298,7 +1301,6 @@ export default class TradeStore extends BaseStore {
             } else {
                 this.validation_rules.duration.rules.push(['number', duration_options]);
             }
-            this.validateProperty('duration', this.duration);
```

Comment:
Why are we removing this validation function?

Reply:
My bad 😅, was trying something with that function. It seems that I accidentally remove the existing ones aswell. Will add it back on the next commit 🙏🏼